sdk/batch/azure-compute-batch/src/main/java/com/azure/compute/batch/models/OptionsBag.java

```diff
@@ -0,0 +1,369 @@
+package com.azure.compute.batch.models;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+
+import com.azure.core.http.RequestConditions;
+
+/**
+ * Represents a bag of optional parameters that can be used to configure various operations.
+ */
```

Comment:
The following properties are body parameters so they shouldn't be included in the options Bag:
- BatchJobTerminateOptions
- endtime
- nodeDisableSchedulingOptionsBody
- nodeRebootOptionsBody
- nodeReimageOptionsBody

Reply:
What about starttime?